Runtime/HighLevel/AccessibleTracker.cs

```diff
@@ -56,6 +56,17 @@ public enum AccessibleType
 
         protected override Dictionary<Enum, string> ExtensionIds => null;
 
+        protected enum ContextActivity
+        {
+            SeriousGames
+        }
+
+        protected Dictionary<Enum, string> contextIds = new Dictionary<Enum, string>()
```

Comment:
This code is repeated across multiple trackers. Consider including it in Abstract High Level Tracker or create a middle class SeriousGameAbstractHighLevelTracker?